portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,22 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> messages = new ArrayList<String>();
+    messages.add("Hello World!");
+    messages.add("Bonjour!");
+    messages.add("Hola!");
+    messages.add("Aloha!");
+
+    // Convert ArrayList to an Object to Json.
+    Data data = new Data(messages);
+    Gson gson = new Gson();
+    String json = gson.toJson(data);
+    
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Matthew Lam!</h1>");
+   // response.getWriter().println("<h1>Hello Matthew Lam!</h1>");
```

Comment:
Because Git stores our complete history, you can just remove the line instead of commenting it out. If you need to restore it later, you will be able to find it in the git history (I can show you this and other great git features later if you're interested). This way, your codebase stays much cleaner instead of accumulating a bunch of commented out code.